### PR TITLE
[SDK-1962] Added Stale

### PR DESCRIPTION
**Pull request by nsingh-branch**
## Reference
SDK-1962 -- Setup "Stale" to automatically close old Github Issues

## Summary
Added a configuration YAML for the Stale GitHub Action. 

## Motivation
Help manage incoming and existing GitHub issues. 

## Type Of Change

- [x] New feature (non-breaking change which adds functionality)

## Testing Instructions

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

**Comment by gdeluna-branch** [comment]
If I read correctly, this applies to issues created after May 22 right? All prior will be spared and we will close them manually.

**Comment by nsingh-branch** [comment]
> If I read correctly, this applies to issues created after May 22 right? All prior will be spared and we will close them manually.

Correct, and I was thinking we could change this based on the repo and its issues. What would you recommend for Android?

**Comment by gdeluna-branch** [comment]
> 

I think that's fine for now, we'll probably want to shorten the window later once things are smoother and in place.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 60
+        days-before-close: 7
+        stale-issue-message: 'This issue has been automatically marked as stale due to inactivity for 60 days. If this issue is still relevant, please respond with any updates or this issue will be closed in 7 days. If you believe this is a mistake, please comment to let us know. Thank you for your contributions.'        
+        stale-issue-label: 'no-issue-activity'
+        close-issue-message: 'This issue has been closed due to inactivity. If this issue is still relevant, please reopen it or create a new one. Thank you for your contributions.'
```
Comment (gdeluna-branch) [inline .github/workflows/stale.yml:27]: Do we have any other stale bots in our organization, what message do they use?

Comment (nsingh-branch) [reply .github/workflows/stale.yml:27]: Not that I know of. This is just the default stale message.

**Pull request merged**